zcash_client_sqlite/src/wallet.rs

```diff
@@ -689,27 +697,17 @@ pub(crate) fn get_wallet_summary(
             }
         };
 
-        account_balances
-            .entry(account)
-            .and_modify(|bal| {
-                bal.sapling_balance.spendable_value = (bal.sapling_balance.spendable_value
-                    + spendable_value)
-                    .expect("Spendable value cannot overflow");
-                bal.sapling_balance.change_pending_confirmation =
-                    (bal.sapling_balance.change_pending_confirmation + change_pending_confirmation)
-                        .expect("Pending change value cannot overflow");
-                bal.sapling_balance.value_pending_spendability =
-                    (bal.sapling_balance.value_pending_spendability + value_pending_spendability)
-                        .expect("Value pending spendability cannot overflow");
-            })
-            .or_insert(AccountBalance {
```

Comment:
Non-blocking: we could probably also remove the `or_insert` from the transparent part.

Reply:
Fixed.